spec/controllers/teams_controller_spec.rb

```diff
@@ -41,6 +41,24 @@
       end
     end
 
+    context 'past teams' do
+      it 'return index by season year' do
+        get :index, params: { year: '2016'}
+        expect(response).to render_template 'index'
+      end
+
+      it 'return teams from current season' do
+        create_list :team, 2, season: Season.current
+        get :index, params: { year: Season.current.name }
+        expect(assigns(:teams).count).to eq 2
+      end
+
+      it 'return empty list when the season is empty of teams' do
+        get :index, params: { year: '2000' }
+        expect(assigns(:teams)).to match_array []
```

Comment:
Hm, not sure if this test is really necessary. Since there aren't any `Team` records in the database, this would be positive for basically any query on the `team` table. I would suggest to either completely skip it, or add some records to really prove this one case.

Reply:
Any news on what I wrote ☝️ @juuh42dias ?

One option could also be: move setting up the teams into a `let` macro
```ruby
let!(:teams) { create_list :team, 2, season: Season.current }
```

Doing so, the test around year 2000 is a bit more meaningful - what do you think?